setup.py

```diff
@@ -10,7 +10,7 @@ def get_long_description():
     long_description = ""
 
     for d in dirs:
-        rst_readme = os.path.join(d, "README.rst")
+        rst_readme = os.path.join(d, "README.md")
```

Comment:
@RamanjaneyuluIdavalapati @prasannababuAddagiri  Why this change? It appears the code is looking for an "rst" file.

Reply:
As discussed, we will remove long_description.